runtime/src/lib.rs

```diff
@@ -228,7 +226,7 @@ impl frame_system::Config for Runtime {
     /// The aggregated dispatch type that is available for extrinsics.
     type Call = Call;
     /// The weight of database operations that the runtime can invoke.
-    type DbWeight = RocksDbWeight;
+    type DbWeight = ();
```

Comment:
why not `RocksDbWeight` instead of `()` like in substrate-parachain-template https://github.com/substrate-developer-hub/substrate-parachain-template/blob/master/runtime/src/lib.rs#L218 and in our standalone chain https://github.com/DataHighway-DHX/node/pull/215/files#diff-0ec06ea58bd455f09ce6b3bb4c2c1c0d37bda51c1e1be2151c560c9c973959ecR284

Reply:
@ltfschoen Since I updated to Polkadot 0.9.9, so I followed 0.9.9 template code base. Here https://github.com/substrate-developer-hub/substrate-parachain-template/blob/polkadot-v0.9.9/runtime/src/lib.rs#L223 it is `()`

Reply:
In Polkadot 0.9.10, they updated it to `RocksDbWeight`.

Reply:
ok we'll stick to updating to 0.9.9 in this PR